### ISSUE TITLE
future work: delete "active flow" collection after flow ends

### ISSUE THREAD

**Issue by robludwig** [cloudmesh/cloudmesh-flow#25]
As explained in the DB section of the manual, we have a new collection when users run a workflow, which we overwrite every time users run that workflow. We should consider 

- summarizing the status of the collection when the flow finishes and then
- deleting the collection to save space

**Comment by robludwig** [comment]
In fact, we could probably harmonize the database a little more. We really only need one workflow collection for definitions and one for active, with a discriminator key called "workflow" that says which node each one is related to. This would need some work if we wanted to be able to run multiple flows at once though. Creating the active collection uses mongodb's `$out` feature which creates a collection and always overwrites it

**Comment by laszewsk** [comment]
Remember at one point we will run multiple workflows at the same time

**Comment by robludwig** [comment]
yes, the current setup supports that but is inefficient. Future work could make it more efficient as described above, but would have to preserve this functionality
